rdopkg_tar/tar_changes.py

```diff
@@ -123,6 +123,16 @@ def commit_distgit_amend(suffix):
     cmd = ['commit', '-a', '-F', '-', '--amend']
     git(*cmd, input=message, print_output=True)
 
+def check_gitlabuser():
+    """
+    Check for the original commiter username in jenkins env variables.
+    If it exists, then amend a suffix to the commit.
+    """
+    userName = os.environ.get('gitlabUserName')
+    gitlabuserName = os.environ.get('gitlabUserUsername')
+
+    if (userName and gitlabuserName):
```

Comment:
Please keep this behavior the same too:
```suggestion
    if userName or gitlabuserName:
```